### PR TITLE
Arrow: Enable python support

### PR DESCRIPTION
**Pull request by singiamtel**

### CODE REVIEW

File: arrow.sh [hunk 1]
```diff
@@ -99,19 +100,21 @@ cmake ./src_tmp/cpp
...
-      -DARROW_PYTHON=OFF                                                                            \
       -DARROW_TENSORFLOW=ON                                                                         \
       -DARROW_GANDIVA=ON                                                                            \
       -DARROW_COMPUTE=ON                                                                            \
+      -DARROW_DATASET=ON                                                                            \
+      -DARROW_FILESYSTEM=ON                                                                         \
       -DARROW_BUILD_STATIC=OFF                                                                      \
       -DCMAKE_INSTALL_RPATH_USE_LINK_PATH=ON                                                        \
-      -DCLANG_EXECUTABLE="$CLANG_EXECUTABLE"
+      -DCLANG_EXECUTABLE="$CLANG_EXECUTABLE"                                                        \
```
Comment (ktf) [inline arrow.sh:117]: Why the extra `\`?

File: arrow.sh [hunk 2]
```diff
@@ -99,19 +100,21 @@ cmake ./src_tmp/cpp
       ${UTF8PROC_ROOT:+-Dutf8proc_ROOT="$UTF8PROC_ROOT"}                                            \
       ${OPENSSL_ROOT:+-DOpenSSL_ROOT="$OPENSSL_ROOT"}                                               \
       ${CLANG_ROOT:+-DLLVM_DIR="$CLANG_ROOT"}                                                       \
+	  ${PYTHON_ROOT:+-DPython3_EXECUTABLE="$PYTHON_ROOT/bin/python3"}								\
```
Comment (ktf) [inline arrow.sh:103]: Please, align things as they already are.

**Review by ktf — CHANGES_REQUESTED** [review]
Ready to go after you fix the nitpicks. Thanks!

File: arrow.sh (hunk 2 above)
Comment (singiamtel) [reply arrow.sh:103]: Oh my bad, I didn't notice that I was aligning with tabs

File: arrow.sh (hunk 1 above)
Comment (singiamtel) [reply arrow.sh:117]: Broken copypaste, will fix

**Comment by singiamtel** [comment]
> Ready to go after you fix the nitpicks. Thanks!

I think it still has some issues on the Mac build, I'd wait for the CI to confirm if it's fixed

**Review by ktf — APPROVED** [review]

**Comment by ktf** [comment]
Currently potentially broken. Merging.

**Pull request merged**